aif360/sklearn/metrics/metrics.py

```diff
@@ -344,6 +344,27 @@ def specificity_score(y_true, y_pred, *, pos_label=1, sample_weight=None,
     return _prf_divide(tn, negs, 'specificity', 'negative', None,
                        ('specificity',), zero_division).item()
 
+def false_omission_score(y_true, y_pred, *, pos_label=1, sample_weight=None,
```

Comment:
By [scikit-learn convention](https://scikit-learn.org/stable/modules/model_evaluation.html#defining-your-scoring-strategy-from-metric-functions), functions ending in `_score` should be maximized while functions ending in `_error` should be minimized. Can we change this name to `false_omission_rate_error`?